test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -67,8 +67,8 @@ if [ ! ${ZCC_TEST_NAME[$FRAMEWORK]} ]; then
     exit 1
 fi
 if [[ ${FRAMEWORK} == "tensorflow2" ]]; then
-    if [ "$(printf '%s\n' "0.7.2" "$SMDEBUG_RELEASE_TAG" | sort -V | head -n1)" = "0.7.2" ]; then
-        timeout 720s python mock/tests/zero_code_change/${FRAMEWORK}_gradtape_integration_tests.py 2>&1
+    if [ "$(printf '%s\n' "0.8.1" "$SMDEBUG_RELEASE_TAG" | sort -V | head -n1)" = "0.8.1" ]; then
```

Comment:
refer to this PR change https://github.com/aws/deep-learning-containers/pull/200/files#diff-6bcfd2946adc5ebde61805459f32293b.

Reply:
Thanks